js/Localization.js

```diff
@@ -0,0 +1,114 @@
+import i18n from 'i18n-js';
+import * as RNLocalize from 'react-native-localize';
+
+i18n.translations = {
+  en: {
+    add_first_site: '+ Add your first site',
+    add_site: 'Add Site',
+    add_sites: 'Add Discourse sites to keep track of.',
+    add: '+ Add',
+    all: 'All',
+    and: 'and',
+    approved_commits: '%{count} commits approved',
+    approved: '"%{title}" approved',
+    check_out_popular: 'Check out these popular communities.',
+    connect: 'connect',
+    dont_know_where_to_start: 'Don’t know where to start?',
+    inbox_message: '%{inbox_count} %{messages} in your %{group_name} inbox',
+    incorrect_url:
+      'Sorry, %{term} is not a correct URL to a Discourse forum or does not support mobile APIs, have owner upgrade Discourse to latest!',
+    liked: 'liked %{count} of your posts',
+    membership_accepted: 'Membership accepted in "%{name}"',
+    message: 'message',
+    messages: 'messages',
+    new_with_count: 'new (%{count})',
+    new: 'New',
+    no_notifications: 'No notifications.',
+    no_replies: 'No replies.',
+    no_sites_yet: 'You don’t have any sites yet.',
+    no_new_notifications: 'No new notifications.',
+    not_found: '%{term} was not found!',
+    notifications: 'Notifications',
+    other: 'other',
+    others: 'others',
```

Comment:
Manual pluralization? You don't have to use `%{count}` in the translation in order to use the pluralization feature of `I18n.t`. It should work if you split the string into `one` and `other` and supply a `count` variable to `I18n.t`